### DISCUSSION TITLE
GDPR banner

### DISCUSSION THREAD

**Discussion by bellons91** [Lednerb/bilberry-hugo-theme#518] [Q&A]
One of the configurations we can set in the `config.toml` file is `googleAnalytics`.

Hoq can we integrate a GDPR banner on the site? I found solutions that "block" the execution of a js script stored in the static/js folder, but that solution does not apply to this theme.

Any idea?

**Comment by bellons91** [comment]
Closed because it was actually described (I was looking for GDPR and not Cookie Banner)

**Comment by igor-baiborodine** [comment]
@bellons91, the header was updated in the README: 

https://github.com/Lednerb/bilberry-hugo-theme#cookie-disclaimer-gdpr

![image](https://user-images.githubusercontent.com/1027701/227933803-85b99c44-a451-4f56-81ef-7a261c865318.png)
